tfhe/src/core_crypto/fft_impl/fft64/crypto/bootstrap.rs

```diff
@@ -525,3 +526,42 @@ where
         );
     }
 }
+
+pub struct BootstrapKeyConformanceParams {
+    pub decomp_base_log: DecompositionBaseLog,
+    pub decomp_level_count: DecompositionLevelCount,
+    pub input_lwe_dimension: LweDimension,
+    pub output_glwe_size: GlweSize,
+    pub polynomial_size: PolynomialSize,
+    pub ciphertext_modulus: CiphertextModulus<u64>,
+}
+
+impl<C: Container<Element = c64>> ParameterSetConformant for FourierLweBootstrapKey<C> {
+    type ParameterSet = BootstrapKeyConformanceParams;
+
+    fn is_conformant(&self, parameter_set: &Self::ParameterSet) -> bool {
+        let Self {
+            fourier:
+                FourierPolynomialList {
+                    data,
+                    polynomial_size,
+                },
+            input_lwe_dimension,
+            glwe_size,
+            decomposition_base_log,
+            decomposition_level_count,
+        } = self;
+
+        data.container_len()
+            == polynomial_size.to_fourier_polynomial_size().0
+                * glwe_size.0
+                * glwe_size.0
+                * input_lwe_dimension.0
+                * decomposition_level_count.0
```

Comment:
do we have a formula for this ? probably not I'm guessing, not a blocker nor necessary

Reply:
Yes, `fourier_ggsw_ciphertext_size`
Replaced